contracts/strategies/1inch/base/OneInchStrategy_DAI_X.sol

```diff
@@ -0,0 +1,300 @@
+pragma solidity 0.5.16;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "../interface/IMooniswap.sol";
+import "../interface/IFarmingRewardsV2.sol";
+
+import "../../../base/interface/IStrategy.sol";
+import "../../../base/upgradability/BaseUpgradeableStrategyUL.sol";
+import "../../../base/interface/IVault.sol";
+
+import "../../../base/StrategyBase.sol";
+
+/**
+* This strategy is for DAI / X 1inch LP tokens
+* DAI must be token0, and the other token is denoted X
+*/
+contract OneInchStrategy_DAI_X is IStrategy, BaseUpgradeableStrategyUL {
+
+  using SafeERC20 for IERC20;
+  using SafeMath for uint256;
+
+  address public constant dai = address(0x6B175474E89094C44Da98b954EedeAC495271d0F);
+  address public constant weth = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+
+  uint256 internal constant maxUint = uint256(~0);
+  uint256 public constant slippageNumerator = 9;
+  uint256 public constant slippageDenominator = 10;
+
+  // depositToken0 is DAI
+  // additional storage slots (on top of BaseUpgradeableStrategy ones) are defined here
+  bytes32 internal constant _DEPOSIT_TOKEN_SLOT = 0x219270253dbc530471c88a9e7c321b36afda219583431e7b6c386d2d46e70c86;
+
+  address[] public rewardTokens;
+
+  constructor() public BaseUpgradeableStrategyUL() {
+    assert(_DEPOSIT_TOKEN_SLOT == bytes32(uint256(keccak256("eip1967.strategyStorage.depositToken")) - 1));
+  }
+
+  function initializeBaseStrategy(
+    address _storage,
+    address _underlying,
+    address _vault,
+    address _rewardPool,
+    uint256 _profitSharingNumerator
+  ) public initializer {
+
+    BaseUpgradeableStrategyUL.initialize(
+      _storage,
+      _underlying,
+      _vault,
+      _rewardPool,
+      weth,
+      _profitSharingNumerator,  // profit sharing numerator
```

Comment:
So just replace this line with line 31 from `OneInchStrategyMainnet_DAI_LDO.sol`

Reply:
Done, and reran test to confirm.